src/main.cairo

```diff
@@ -1,32 +1,182 @@
 %lang starknet
 from starkware.cairo.common.math import assert_nn
 from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.starknet.common.syscalls import get_caller_address, get_block_timestamp, get_contract_address
+from starkware.cairo.common.math import assert_not_zero, assert_le
+
+from lib.cairo_contracts.src.openzeppelin.token.erc20.IERC20 import IERC20
+from src.interface.naming import Naming
+from src.interface.pricing import Pricing
+
+//
+// Storage
+//
+
+@storage_var
+func naming_contract() -> (contract_address: felt) {
+}
+
+@storage_var
+func pricing_contract() -> (contract_address: felt) {
+}
 
 @storage_var
-func balance() -> (res: felt) {
+func renews(renewer: felt, domain: felt) -> (bool: felt) {
+}
+
+@storage_var
+func voted_upgrade(user: felt, upgrade: felt) -> (bool: felt) {
+}
+
+//
+// Events
+//
+
+@event
+func ToggledRenewal(domain: felt, renewer: felt, value: felt) {
```

Comment:
Why do events have uppercase and camel case ? When others are in lowercase and snake case :)

I don't know if it's a convention or just a liberty that you took :)

Reply:
I changes to use your convention